### PR TITLE
Adds Office world

### PR DESCRIPTION
**Pull request by Voldivh**
# 🎉 New Map

## Summary
This PR adds a new world to the simulation (Office).

## Checklist
- [X] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)

### CODE REVIEW

**Comment by francocipollone** [comment]
@Voldivh I just pushed some env hooks for hooking the IGN_RESOURCE_PATH env var so the models can be found

File: andino_gz/models/magazino_hallway.stl
Comment (francocipollone) [inline andino_gz/models/magazino_hallway.stl:1]: I think that this is for other kind of world: Magazino's dataset like

File: andino_gz/worlds/office.world
Comment (francocipollone) [inline andino_gz/worlds/office.world:1]: for consistency, name it at office.sdf

File: andino_gz/worlds/office.world [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+              <normal>0 0 1</normal>
+              <size>100 100</size>
+            </plane>
+          </geometry>
+          <material>
+            <script>
+              <uri>file://media/materials/scripts/gazebo.material</uri>
+              <name>Gazebo/Grey</name>
+            </script>
+          </material>
```
Comment (francocipollone) [inline andino_gz/worlds/office.world]: script materials aren't supported in the Gazebo non classic

File: andino_gz/models/office/model.sdf [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <uri>model://office.stl</uri>
+            <scale>1 1 1</scale>
+          </mesh>
+        </geometry>
+        <material>
+          <lighting>1</lighting>
+          <script>
+            <uri>file://media/materials/scripts/gazebo.material</uri>
+            <name>Gazebo/Grey</name>
+          </script>
```
Comment (francocipollone) [inline andino_gz/models/office/model.sdf:35]: script material can't be used in this gazebo version

**Review by francocipollone — CHANGES_REQUESTED** [review]
Note that model and world need to be migrated to Gazebo non-classic.
I will see if I can help you out with that.

**Comment by Voldivh** [comment]
@francocipollone Just updated the world, I think the current state works for our purpose on the andino+rmf project and improvements on this map textures and the general world can be addressed on a follow up PR. This is how it looks like atm: 
image

**Review by francocipollone — APPROVED** [review]
Merging to unblock you.

Pendings:
 - https://github.com/Ekumen-OS/andino_gz/issues/59
 - https://github.com/Ekumen-OS/andino_gz/issues/60

**Pull request merged**